tiledb/ml/models/sklearn.py

```diff
@@ -40,13 +46,19 @@ def save(
         if not update:
             self._create_array()
 
-        self._write_array(serialized_model=serialized_model, meta=meta)
+        if self.tiledb_uri and self.ctx:
+            self._write_array(uri=self.tiledb_uri, serialized_model=serialized_model, meta=meta)
+        else:
+            self._write_array(uri=self.uri, serialized_model=serialized_model, meta=meta)
 
     def load(self) -> BaseEstimator:
         """
         Loads a Sklearn model from a TileDB array.
         """
-        model_array = tiledb.open(self.uri)
+        if self.tiledb_uri and self.ctx:
+            model_array = tiledb.open(uri=self.tiledb_uri, ctx=self.ctx)
+        else:
+            model_array = tiledb.open(self.uri)
```

Comment:
Same comment about `tiledb_uri` vs `uri`, plus `ctx` is not a tiledb-cloud specific parameter.